plugins/devices/frontend-java/src/main/java/com/freedomotic/jfrontend/MainWindow.java

```diff
@@ -1500,7 +1500,7 @@ private void sendLogByMail() {
     private void jMenuItem1ActionPerformed(java.awt.event.ActionEvent evt) {
         String[] buttons = {i18n.msg("form_compilation"), i18n.msg("send_log"), i18n.msg("delete")};
         int option = JOptionPane.showOptionDialog(null, i18n.msg("report_issue_dialog"), i18n.msg("report_issue"),
-                JOptionPane.WARNING_MESSAGE, 0, null, buttons, buttons[2]);
+                JOptionPane.OK_CANCEL_OPTION, JOptionPane.ERROR_MESSAGE, null, buttons, buttons[2]);
```

Comment:
By the way, this was changed as 3rd parameter of showOptionDialog is about buttons option, and 4th is about the style. I've changed parameters, but values remain the same - 2 and 0.